portfolio/src/main/webapp/script.js

```diff
@@ -70,3 +70,11 @@ function contact() {
 		</ul>`; 
 	console.log('Contact!'); 
 }
+
+async function fetchHelloWorld() {
+	document.getElementById("inner-content").style.display = "block";
+	const response = await fetch('/data');
+	const hello = await response.text();
```

Comment:
nit - can you choose a more descriptive variable name? `helloMessage` or something like that?

Reply:
thanks! will do!